app/models/project.rb

```diff
@@ -346,7 +346,25 @@ def normalize_licenses
     elsif licenses.length > 150
       normalized = ['Other']
     else
-      normalized = licenses.split(/[,\/]/).map do |license|
+      downcased = licenses.downcase
+      # chomp off leading/trailing () to make Spdx.find happier
+      if downcased.start_with?('(')
+        downcased[0] = ''
+      end
+      if downcased.start_with?(')')
+        downcased[-1] = ''
+      end
+      # splits are OR, AND, and COMMA (,)
+      # technically OR and AND are different in meaning
+      # but our model doesn't allow the distinction
+      if downcased.include?("or")
+        split = downcased.split(/or/)
+      elsif downcased.include?("and")
+        split = downcased.split(/and/)
+      else
+        split = licenses.split(/[,\/]/)
```

Comment:
does this leave whitespace around the split pieces? from the tests it looks like it vanishes somehow but maybe it's in the Spdx.find

Reply:
I guess we allow either comma or `/` but the comment doesn't mention that

Reply:
Yeah, Spdx.find is smart enough with whitespace. Fixed the comment